crates/arroyo-connectors/src/iggy/table.json

```diff
@@ -0,0 +1,86 @@
+{
+    "type": "object",
+    "title": "IggyTable",
+    "properties": {
+        "stream": {
+            "title": "Stream",
+            "type": "string",
+            "description": "The Iggy stream to use for this table"
+        },
+        "topic": {
+            "title": "Topic",
+            "type": "string",
+            "description": "The Iggy topic to use for this table"
+        },
+        "type": {
+            "type": "object",
+            "title": "Table Type",
+            "oneOf": [
+                {
+                    "type": "object",
+                    "title": "Source",
+                    "properties": {
+                        "offset": {
+                            "type": "string",
+                            "description": "The offset to start reading from",
+                            "enum": [
+                                "latest",
+                                "earliest"
+                            ]
+                        },
+                        "consumer_id": {
+                            "type": "integer",
+                            "title": "Consumer ID",
+                            "description": "The ID of the consumer to use for polling messages"
+                        },
+                        "partition_id": {
```

Comment:
I'm not sure we want to have the user choose a particular partition to read from; generally you want to read from all partitions. At the very least I think we need an option to read from all partitions.